src/arvgvinterface.c

```diff
@@ -353,8 +360,20 @@ struct _ArvGvInterfaceClass {
 
 G_DEFINE_TYPE_WITH_CODE (ArvGvInterface, arv_gv_interface, ARV_TYPE_INTERFACE, G_ADD_PRIVATE (ArvGvInterface))
 
+void
```

Comment:
Make these two functions use arv_gv_interface_get_instance to find the interface instance. Use the mutex to protect private data access.